lib/llm/src/http/service/service_v2.rs

```diff
@@ -281,7 +281,13 @@ impl HttpService {
             })?;
 
             axum::serve(listener, router)
-                .with_graceful_shutdown(observer.cancelled_owned())
+                .with_graceful_shutdown(async {
+                    observer.cancelled_owned().await;
+                    tracing::info!("HTTP server shutdown requested");
+                    // accepting requests for 5 more seconds, to allow incorrectly routed requests to arrive
```

Comment:
Question: for my understanding - `incorrectly routed requests` - this is from a gateway in front of this service?

Or is it incorrectly routed request from this server to a worker?